src/UniversalRewardsDistributor.sol

```diff
@@ -86,12 +92,14 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
     /// @dev This function can only be called after the timelock has expired.
     /// @dev Anyone can call this function.
     function acceptRoot() external {
-        require(pendingRoot.submittedAt > 0, ErrorsLib.NO_PENDING_ROOT);
-        require(block.timestamp >= pendingRoot.submittedAt + timelock, ErrorsLib.TIMELOCK_NOT_EXPIRED);
+        PendingRoot memory pendingRootMem = pendingRoot;
```

Comment:
harmonize code with `setTimelock` but we could do the opposite

Reply:
this is optimized by default (and refactored into the other sense [here](https://github.com/morpho-org/universal-rewards-distributor/pull/39#discussion_r1328601769)

Reply:
so we should do the opposite

Reply:
ok to change the other way around